tests/nnstreamer_sink/unittest_sink.cc

```diff
@@ -459,559 +475,500 @@ _setup_pipeline (TestOption & option)
+        "videotestsrc num-buffers=%d ! video/x-raw,width=64,height=48,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_2 "
+        "demux. ! queue ! tensor_sink name=test_sink "
+        "demux. ! queue ! tensor_sink",
+        option.num_buffers, option.num_buffers * 3, option.num_buffers + 3);
+    break;
+  case TEST_TYPE_CUSTOM_TENSOR:
+    /** video 160x120 RGB, passthrough custom filter */
+    str_pipeline = g_strdup_printf (
+        "videotestsrc num-buffers=%d ! videoconvert ! video/x-raw,width=160,height=120,format=RGB,framerate=(fraction)%lu/1 ! "
+        "tensor_converter ! tensor_filter name=test_filter framework=custom model=%s/libnnstreamer_customfilter_passthrough_variable%s ! tensor_sink name=test_sink",
+        option.num_buffers, fps, custom_dir ? custom_dir : "./nnstreamer_example",
+        NNSTREAMER_SO_FILE_EXTENSION);
+    break;
+  case TEST_TYPE_CUSTOM_TENSORS_1:
+    /** other/tensors with tensormux, passthrough custom filter */
+    str_pipeline = g_strdup_printf (
+        "tensor_mux name=mux ! tensor_filter name=test_filter framework=custom model=%s/libnnstreamer_customfilter_passthrough_variable%s ! tensor_sink name=test_sink "
+        "videotestsrc num-buffers=%d ! video/x-raw,width=160,height=120,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_0 "
+        "videotestsrc num-buffers=%d ! video/x-raw,width=120,height=80,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_1 "
+        "videotestsrc num-buffers=%d ! video/x-raw,width=64,height=48,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_2",
+        custom_dir ? custom_dir : "./nnstreamer_example", NNSTREAMER_SO_FILE_EXTENSION,
+        option.num_buffers, option.num_buffers, option.num_buffers);
+    break;
+  case TEST_TYPE_CUSTOM_TENSORS_2:
+    /** other/tensors with tensormux, passthrough custom filter */
+    str_pipeline = g_strdup_printf (
+        "tensor_mux name=mux ! tensor_filter name=test_filter framework=custom model=%s/libnnstreamer_customfilter_passthrough_variable%s "
+        "input=3:160:120:1,3:120:80:1,3:64:48:1 output=3:160:120:1,3:120:80:1,3:64:48:1 inputtype=uint8,uint8,uint8 outputtype=uint8,uint8,uint8 "
+        "inputlayout=NCHW,NHWC,NONE outputlayout=ANY,NHCW,NCHW ! tensor_sink name=test_sink "
+        "videotestsrc num-buffers=%d ! video/x-raw,width=160,height=120,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_0 "
+        "videotestsrc num-buffers=%d ! video/x-raw,width=120,height=80,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_1 "
+        "videotestsrc num-buffers=%d ! video/x-raw,width=64,height=48,format=RGB,framerate=(fraction)30/1 ! tensor_converter ! mux.sink_2",
+        custom_dir ? custom_dir : "./nnstreamer_example", NNSTREAMER_SO_FILE_EXTENSION,
+        option.num_buffers, option.num_buffers, option.num_buffers);
+    break;
+  case TEST_TYPE_CUSTOM_MULTI:
+    /* multiple custom filters */
```

Comment:
Q) `/** COMMENT */` and `/* COMMENT */` what is the right way for the single line comment? there are both of them.

Reply:
Single line: both are good
Multi line or Doxygen: ```/** */ ```